runelite-client/src/main/java/net/runelite/client/plugins/achievementdiary/diaries/LumbridgeDiaryRequirement.java

```diff
@@ -56,33 +62,47 @@ public LumbridgeDiaryRequirement()
 			new Requirement(Skill.STRENGTH, 19),
 			new Requirement(Skill.RANGED, 37));
 		add("Purchase an upgraded device from Ava.",
-			new Requirement(Skill.RANGED, 50));
+			new Requirement(Skill.RANGED, 50),
+			new Requirement(Quest.ANIMAL_MAGNETISM));
+		add("Ttravel to the Wizards' Tower by Fairy ring.",
+			new Requirement(Quest.FAMILY_CREST, true));
 		add("Cast the teleport to Lumbridge spell.",
 			new Requirement(Skill.MAGIC, 31));
 		add("Catch some Salmon in Lumbridge.",
 			new Requirement(Skill.FISHING, 30));
 		add("Craft a coif in the Lumbridge cow pen.",
 			new Requirement(Skill.CRAFTING, 38));
-		add("Get a slayer task from Chaeldar.",
-			new Requirement("Combat", 70));
 		add("Chop some willow logs in Draynor Village.",
 			new Requirement(Skill.WOODCUTTING, 30));
 		add("Pickpocket Martin the Master Gardener.",
 			new Requirement(Skill.THIEVING, 38));
+		add("Get a slayer task from Chaeldar.",
+			new Requirement("Combat", 70),
+			new Requirement(Quest.LOST_CITY));
 		add("Catch an Essence or Eclectic impling in Puro-Puro.",
-			new Requirement(Skill.HUNTER, 42));
+			new Requirement(Skill.HUNTER, 42),
+			new Requirement(Quest.LOST_CITY));
 		add("Craft some Lava runes at the fire altar in Al Kharid.",
-			new Requirement(Skill.RUNECRAFT, 23));
+			new Requirement(Skill.RUNECRAFT, 23),
+			new Requirement(Quest.RUNE_MYSTERIES));
```

Comment:
You can do tasks out of order, so it would be best to keep it there.

Reply:
Pretty sure you don't actually need to complete rune mysteries to runecraft?

Reply:
According to the wiki, you do. If you can prove otherwise, I would be happy to remove them as requirements. I don't have an account to test this on unfortunately.

Reply:
If that's the case, I'm fairly certain there are some other tasks requiring runecrafting which don't explicitly call out the rune mysteries quest requirement.

Reply:
As the wiki mentions Rune Mysterious being a requirement for the tasks, I'm going to leave it in until someone is able to complete a runecraft task without having the quest complete. I unfortunately do not have an account to test this with.